todel/src/models/info.rs

```diff
@@ -3,7 +3,7 @@ use serde::{Deserialize, Serialize};
 
 /// The instance info payload
```

Comment:
Should these end with a `.`?

Reply:
Yup, that'll be added along with the rest of the documentation.